vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -60,16 +60,17 @@ $global:defenderUpdateInfoUrl = "https://go.microsoft.com/fwlink/?linkid=870379&
 
 switch -Regex ($windowsSku) {
     "2019-containerd" {
-        # TODO: clean up when you bump 12B. These patches do an upgrade from 10B to 12B in a way that doesn't break openssh.
```

Comment:
Does you still hit the openssh permissions problem with WS2019 base image with 12b?

Reply:
I sure did! Was super annoying.